app/src/main/java/com/android/streetworkapp/ui/tutorial/TutorialEvent.kt

```diff
@@ -0,0 +1,214 @@
+package com.android.streetworkapp.ui.tutorial
+
+import android.util.Log
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.pager.HorizontalPager
+import androidx.compose.foundation.pager.rememberPagerState
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.text.SpanStyle
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.withStyle
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.android.sample.R
+import com.android.streetworkapp.ui.navigation.NavigationActions
+import com.android.streetworkapp.ui.navigation.Screen
+import com.android.streetworkapp.ui.theme.ColorPalette
+import com.android.streetworkapp.ui.theme.DarkGray
+import com.android.streetworkapp.ui.theme.GoogleAuthButtonTextStyle
+import com.android.streetworkapp.ui.theme.White
+
+@Composable
+fun TutorialEvent(navigationActions: NavigationActions) {
+  // Set number of page
+  val pagerState = rememberPagerState(pageCount = { 5 })
+
+  // Tutorial text for each page
+  val tutoText1 = buildAnnotatedString {
+    withStyle(style = SpanStyle(color = ColorPalette.TUTORIAL_INTERACTION_1)) { append("Click") }
+    append(" on your favorite park")
+  }
+
+  val tutoText2 = buildAnnotatedString {
+    withStyle(style = SpanStyle(ColorPalette.TUTORIAL_INTERACTION_2)) { append("Look") }
+    append(" for others events in the park\nOr ")
```

Comment:
Typo: there is a small typo here (it should be "other" without the s)

Reply:
putted "Look for other people's events in the park"